code/modules/mob/living/living_defense.dm

```diff
@@ -154,6 +154,7 @@
 	if(!on_fire)
 		return FALSE
 	on_fire = FALSE
+	adjust_bodytemperature(bodytemperature - 80, 80)
```

Comment:
``min_temp`` is set to 80 via parameter, making it possible to freeze a marine, is that intended?

Reply:
The lower bound shouldnt matter, in theory, but I'll set it to 300 anyway